components/automate-deployment/habitat/plan.sh

```diff
@@ -26,6 +26,7 @@ pkg_deps=(
   core/rsync
   core/tar
   chef/mlsa
+  core/postgresql11-client
```

Comment:
I feel like a comment here or _somewhere_ reminding us why we are on this version might be good.